inference-engine/src/mkldnn_plugin/emitters/jit_load_store_emitters.cpp

```diff
@@ -509,6 +509,12 @@ size_t jit_store_emitter::aux_vecs_count() const {
 
 size_t jit_store_emitter::get_inputs_num() const { return 1; }
 
+void jit_store_emitter::emit_data() const {
+    jit_emitter::emit_data();
```

Comment:
I think we can remove this function call that do noting, given no data need emitted for jit_emitter or jit_store_emitter itself currently as entry_map_ is empty, just need emit_data() for emu_vcvtneps2bf16.